MYabrv.bib

```diff
@@ -309,7 +324,8 @@ @String{Piscataway
 	@String{Miami = "Miami, FL, USA"}
 	@String{Montreal = "Montreal, Quebec, Canada"}
 	@String{LosAlamitos = "Los Alamitos, CA, USA"}
-	@String{GI = "Gesellschaft f{\"u}r Informatik"}
+
+@String{GI = "Gesellschaft f{\"u}r Informatik e.V."}
```

Comment:
Have never seen this before in a literature entry. Please revert.

Reply:
I have seen it in multiple entries and also it is the only "legal" spelling of the organisation, but will revert.